src/components/Filter/FilterItem.js

```diff
@@ -0,0 +1,38 @@
+import cx from 'classnames';
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const FilterItem = ({ className, label, onRemove, filterData, ...rest }) => {
+  const classes = cx(className);
```

Comment:
For some reason I keep coming back to this bit... 
```js
FilterItem = ({ className, label, onRemove, filterData, ...rest })
```
Not sure if we do this anywhere else within this repo, but ```label``` is a property on ```filterdata``` is that something we could combine to avoid passing the same data in while relating some of the context?

Reply:
Yeah, we would have to force the filterData structure. And the label property is being removed in favor of children to allow users to show more than just a label here.